pipeline/objects/__init__.py

```diff
@@ -23,12 +22,13 @@ def execute_func(*args, **kwargs):
                 if isinstance(input_arg, Variable):
                     processed_args.append(input_arg)
                 elif hasattr(input_arg, "__pipeline_model__"):
-                    if function.__pipeline_function__.class_instance == None:
+                    if function.__pipeline_function__.class_instance is None:
                         function.__pipeline_function__.class_instance = input_arg
 
                 else:
                     raise Exception(
-                        "You can't input random variables, follow the way of the Pipeline. Got type"
+                        "You can't input random variables,",
+                        "follow the way of the Pipeline. Got type",
```

Comment:
Same comma comment here. We could add the type to the message while we're here; looks like it should show `type(input_arg)`.